lib/health/account/user.ex

```diff
@@ -1,15 +1,31 @@
 defmodule Health.Account.User do
   @moduledoc false
+  alias Health.UserPolicy
   use Ecto.Schema
   use Pow.Ecto.Schema
+  import Ecto.Changeset
 
   schema "users" do
     has_many :logs, Health.Weight.Log
+    field :roles, :integer
     pow_user_fields()
 
     timestamps()
   end
 
+  defdelegate authorize(action, user, params), to: UserPolicy
+
+  @spec changeset(
+          {map, map} | %{:__struct__ => atom | %{__changeset__: map}, optional(atom) => any},
+          :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
+        ) :: Ecto.Changeset.t()
+  def changeset(user, attrs) do
+    user
+    |> cast(attrs, [:roles])
+    |> validate_required([:roles])
+    |> validate_number(:roles, greater_than_or_equal_to: 0, less_than: 11)
+  end
```

Comment:
The only changeset allowed...

Reply:
I am considering talking to the pow maintainer and maybe working on a pull request to enable using multiple changesets